MLDSAI/OpenAdapt
# Github Actions for automatic linting/testing/publishing

## Overview

- Resolves #22 
- Using Github Actions script that reformats code with python black, and runs linters and pytests whenever you push to any branch

## Unit Tests Created

- n/a

## Steps to QA

- See Github Actions run history on my repo: https://github.com/Mustaballer/puterbot/actions
